pom.xml

```diff
@@ -189,6 +193,33 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
+          <version>1.1.11</version>
+          <configuration>
+            <reportsDirectory>local/pit-reports</reportsDirectory>
+            <timestampedReports>false</timestampedReports>
+            <timeoutConstant>1000</timeoutConstant>
+            <targetClasses>
+              <param>${pitest.classes}</param>
```

Comment:
Reading this
  http://pitest.org/quickstart/maven/
I wonder if -DtargetClasses or -Dpit.targetClasses doesn't work, because all "extra" properties do have command line switch, so why wouldn't the "non-extra" one have it?
If not, an issue needs to be raised in their issue tracker.

Either way, I 'd prefer to remove that module specific property "pitest.classes", even if that disabled the command line override for now... because that's a maintenance pain (on drools, optaplanner, optaplanner-wb, ...) as well as a distraction (less in the pom is more - pitest should take up pom space except in the parent). Wydt?